website/docs/cdktf/create-and-deploy/deployment-patterns.mdx

```diff
@@ -0,0 +1,169 @@
+
+### Run `cdktf synth` and Use Terraform Directly
+
+We recommend synthesizing your CDKTF application and using Terraform directly when you need to integrate into existing Terraform workflows. Your organization may already use Terraform Cloud or have integrated Terraform into CI pipelines. In these cases, you may want to run `cdktf synth` and set the output folder to a particular directory that you can reference in your existing workflows.
+
+## Using Terraform Cloud
+
+Terraform Cloud lets you review the current state of your infrastructure and run progress in the Terraform Cloud UI. You can also use Terraform Cloud as the source of secrets via [workspace variables](/cloud-docs/workspaces/variables). If you use the local execution mode or any other backend you must specify the credentials for your providers locally through environment variables.
+
+To use Terraform Cloud with a [CLI-driven Run Workflow](/cloud-docs/run/cli), you must enable [remote execution](/cloud-docs/run#remote-operations) in your Terraform Cloud workspace.
+
+~> **Warning:** Remote execution with non-interactive workflows requires auto-approved deployments. Minimize risk of unpredictable infrastructure changes and configuration drift by making sure that no one can change your infrastructure outside of your automated build pipeline. Refer to [Non-Interactive Workflows](/cloud-docs/run/cli#non-interactive-workflows) for details.
+
+To run Terraform Cloud with a [VCS-Driven Run Workflow](/cloud-docs/run/ui) you must check in your synthesized code.
+
+## Managing Secrets and Sensitive Data
+
+No matter if you manage your secrets through Terraform Cloud (in `remote-execution` mode) or through your normal CI, secrets should never appear in your synthesized code. [This is why you should read Secrets with Terraform Variables](/cdktf/create-and-deploy/best-practices#read-secrets-with-terraform-variables).
+
+## Example Workflows
+
+### Deploy with Terraform and Terraform Cloud
+
+You could create a workflow with the following steps:
+
+1. Add a pre-commit step that runs `cdktf synth`. This creates Terraform configuration files under `cdktf.out/stacks/<stack-name>`.
+2. Configure the VCS integration for a Terraform cloud workspace to point to the directory with the synthesized configuration files. To deploy multiple independent stacks, configure one Terraform Cloud workspace to point to the output directory for each stack.
+
+Refer to [UI/VCS-Driven Run Workflow](/cloud-docs/run/ui) in the Terraform Cloud documentation for details about how to trigger Terraform runs from merges and commits.
+
+### Terraform CDK GitHub Action
+
+If you are using GitHub Actions and you want a default review / deployment workflow, you can use the [Terraform CDK GitHub Action](https://github.com/marketplace/actions/terraform-cdk-action). It provides you with comments on your PRs and a default review workflow. If you need more customization, please take a look at the next example.
+
```

Comment:
Can you say here if folks are synthesizing or running with CDKTF directly? I just wasn't sure I think because it's TFC.